modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/impl/WorkflowServiceImpl.java

```diff
@@ -1176,6 +1190,7 @@ public void update(final WorkflowInstance workflowInstance) throws WorkflowDatab
       }
 
       MediaPackage updatedMediaPackage = null;
+      Opt<AccessControlList> updatedAcl = Opt.none();
```

Comment:
I'm somewhat surprised this passes CI, but `Opt` is not a found symbol here anymore.  Replacing `Opt` with `Option` is a drop-in replacement.